mock/etc/bash_completion.d/mock

```diff
@@ -23,15 +23,35 @@ _mock_root()
     _filedir 'cfg'
 }
 
+_mock_isopt()
+{
+    [[ ${1:0:1} = - ]] && return 0
+    case $1 in
+    install|remove)
+            return 0
+            ;;
+    esac
+    return 1
+}
+
 _mock()
 {
     local cur prev words cword split
     _init_completion -s || return
 
     local cfgdirs=( /etc/mock "$HOME/.config/mock" )
     local count=0
+    local prevopt
```

Comment:
Last nit... sorry for being pedantic, but I noticed this now .... do we need `$prevopt` or can we keep using `$prev`?

Reply:
prev is set and defined by `_init_completion`.  I think it's more clear to use something different.